proto/attestation/endorsement.proto

```diff
@@ -30,6 +30,9 @@ message TransparentReleaseEndorsement {
 
   // The log entry as proof of inclusion of the endorsement statement in Rekor.
   bytes rekor_log_entry = 2;
+
+  // The signature for the endorsement.
```

Comment:
How about:

"The signature for the endorsement, if a log entry is not available."

This clarifies the dependency.

Also, maybe move it above rekor_log_entry (to match the order in which they are generated, rekor_log_entry is the most complex). It is no problem to change the wire numbers at this point, i.e. fine to swap 2 and 3.

Reply:
Reordered the fields but from the offline discussion, sounds like the signature will also be included when the log is available.